resources/controllers/Admin.php

```diff
@@ -210,10 +301,10 @@ public function admin_categories_page()
 
```

Comment:
@Osoro In line [293](https://github.com/zype/zype-wordpress-plugin/pull/38/files?utf8=%E2%9C%93&diff=split#diff-79dfb295029a45fe9eca518c86fbb08eR293) shouldn't we use `$this->options`?